rubocop-rails.gemspec

```diff
@@ -4,6 +4,7 @@ $LOAD_PATH.unshift File.expand_path('lib', __dir__)
 require 'rubocop/rails/version'
 require 'English'
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
Can you specify in the exclusion list of this cop at .rubocop.yml?
https://github.com/rubocop-hq/rubocop-rails/blob/v2.4.2/.rubocop.yml#L80-L84